<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>auth</class>
 <widget class="QMainWindow" name="auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Математический тренажёр</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="leftPanel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #aa0000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="welcome_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Добрый день, пользователь</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabW">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>-25</y>
      <width>611</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	font: 87 14pt &quot;Arial&quot;;
	color: white;
	background-color: #aa0000;
	border: 2px solid white;
	border-radius: 10px;
}
QPushButton:hover {
    background: rgb(125, 0, 0)

    }
QPushButton:pressed {
    border-style: inset;
    background: rgb(85, 85, 85);
    }</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="authLbl">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>201</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Авторизация</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="authBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>330</y>
        <width>180</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	font: 87 14pt &quot;Arial&quot;;
	color: white;
	background-color: #aa0000;
	border: 2px solid white;
	border-radius: 10px;
}
QPushButton:hover {
    background: rgb(125, 0, 0)

    }
QPushButton:pressed {
    border-style: inset;
    background: rgb(85, 85, 85);
    }</string>
      </property>
      <property name="text">
       <string>ВХОД</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="login">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">
	font: 87 14pt &quot;Arial&quot;;
	color: black;
	background-color: #cccccc;
	border: 2px solid grey;
	border-radius: 10px;
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="placeholderText">
       <string>Логин...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>270</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
	font: 87 14pt &quot;Arial&quot;;
	color: black;
	background-color: #cccccc;
	border: 2px solid grey;
	border-radius: 10px;
</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Пароль...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>561</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../data/img/bg.png</pixmap>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>authLbl</zorder>
     <zorder>authBtn</zorder>
     <zorder>login</zorder>
     <zorder>password</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLineEdit" name="class_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
	font: 87 14pt &quot;Arial&quot;;
	color: black;
	background-color: #cccccc;
	border: 2px solid grey;
	border-radius: 10px;
</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Класс...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="regBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>420</y>
        <width>180</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	font: 87 14pt &quot;Arial&quot;;
	color: white;
	background-color: #aa0000;
	border: 2px solid white;
	border-radius: 10px;
}
QPushButton:hover {
    background: rgb(125, 0, 0)

    }
QPushButton:pressed {
    border-style: inset;
    background: rgb(85, 85, 85);
    }</string>
      </property>
      <property name="text">
       <string>РЕГИСТРАЦИЯ</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lastName">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">
	font: 87 14pt &quot;Arial&quot;;
	color: black;
	background-color: #cccccc;
	border: 2px solid grey;
	border-radius: 10px;
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="placeholderText">
       <string>Фамилия...</string>
      </property>
     </widget>
     <widget class="QLabel" name="regLbl">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>201</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>561</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../data/img/bg.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="firstName">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">
	font: 87 14pt &quot;Arial&quot;;
	color: black;
	background-color: #cccccc;
	border: 2px solid grey;
	border-radius: 10px;
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="placeholderText">
       <string>Имя...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginR">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>310</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
	font: 87 14pt &quot;Arial&quot;;
	color: black;
	background-color: #cccccc;
	border: 2px solid grey;
	border-radius: 10px;
</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Логин...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordR">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>360</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
	font: 87 14pt &quot;Arial&quot;;
	color: black;
	background-color: #cccccc;
	border: 2px solid grey;
	border-radius: 10px;
</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
      <property name="placeholderText">
       <string>Пароль...</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>class_2</zorder>
     <zorder>regBtn</zorder>
     <zorder>lastName</zorder>
     <zorder>regLbl</zorder>
     <zorder>firstName</zorder>
     <zorder>loginR</zorder>
     <zorder>passwordR</zorder>
    </widget>
   </widget>
   <widget class="QPushButton" name="reg_switch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	font: 87 14pt &quot;Arial&quot;;
	color: white;
	background-color: #aa0000;
	border: 2px solid white;
	border-radius: 10px;
}
QPushButton:hover {
    background: rgb(125, 0, 0)

    }
QPushButton:pressed {
    border-style: inset;
    background: rgb(85, 85, 85);
    }</string>
    </property>
    <property name="text">
     <string>РЕГИСТРАЦИЯ</string>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="auth_switch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	font: 87 14pt &quot;Arial&quot;;
	color: white;
	background-color: #aa0000;
	border: 2px solid white;
	border-radius: 10px;
}
QPushButton:hover {
    background: rgb(125, 0, 0)

    }
QPushButton:pressed {
    border-style: inset;
    background: rgb(85, 85, 85);
    }</string>
    </property>
    <property name="text">
     <string>АВТОРИЗАЦИЯ</string>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="tech_supportBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	font: 87 14pt &quot;Arial&quot;;
	color: white;
	background-color: #aa0000;
	border: 2px solid white;
	border-radius: 10px;
}
QPushButton:hover {
    background: rgb(125, 0, 0)

    }
QPushButton:pressed {
    border-style: inset;
    background: rgb(85, 85, 85);
    }</string>
    </property>
    <property name="text">
     <string>ПОДДЕРЖКА</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../data/img/telegram.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
